requirements.txt

```diff
@@ -1,13 +1,13 @@
-click
-simplejson
-pyzmq
-PyYaml
-requests
-minio
-pip
-schematics
-aiohttp
-aiohttp-cors
-aiobotocore
-emloop
-apistrap
+click>=7.0
+simplejson>=3.16
+pyzmq>=18.0
+PyYaml>=5.1
```

Comment:
Do we need this? Emloop already has ruamel.

Reply:
sure, but shepherd uses pyyaml on its own...